Luminary099/RTB_OP_CODES.agc

```diff
@@ -167,7 +167,7 @@ PULSEIMU	INDEX	FIXLOC		# ADDRESS OF GYRO COMMANDS SHOULD BE IN X1
 #	2.	GENERATE IN MPAC THE SIGNUM FUNCTION OF MPAC:
 #		ENTRY:		RTB
 #					SIGNMPAC
-# IN EITHER CASE, RETURN IS TO TEH NEXT INTERPRETIVE INSTRUCTION IN THE CALLING SEQUENCE.
```

Comment:
Two spaces between TO and THE ([source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1401.jpg))